src/shared/constants.js

```diff
@@ -1080,7 +1080,12 @@ export const extractData = (attribute, from) => {
   let data;
   let dataType;
   if (from === 'infoSheet') {
-    data = attribute?.values[0]?.label ? attribute?.values[0]?.label : attribute?.values[0]?.value;
```

Comment:
this should already handle it properly right? data gets returned in a typography on line 1120

Reply:
this is still broken but i think it's now an api thing. @trevorjpuckett in the crops endpoint the label is showing up properly but in the info sheet endpoint crop.data[index].attributes.values[0] has label as null for everything

Reply:
@mikahpinegar looks like some are showing up properly, while others are not. this looks like its an error in the dataset. I will have this fixed by EoD today!